avocado/core/nrunner.py

```diff
@@ -500,6 +500,7 @@ def run(self):
             return
 
         queue = multiprocessing.SimpleQueue()
+        sys.path.insert(0, ".")
```

Comment:
Maybe doing this on `_run_unittest()` itself is a better strategy? This way it happens only in the process running the unittest, and not on the "runner" process.

Reply:
Yes, I was in doubt, because it was running for every test, so I decided to move here. But agree, at least will be executed on the subprocess.

Reply:
Since it was a small change, I just force-pushed here.